docs/synctl-update-test.md

```diff
@@ -49,7 +50,6 @@ synctl update test <id> [options]
 
 ### Options for API Script test
 ```
-    -f, --from-file <file-name>        load synthetic test payload from (.json) file
     --script <file>                    load script (.js) from file
     --bundle <bundle>                  set bundle content
```

Comment:
we can make it more clear like "Synthetic bundle test script, support zip file (.zip) path  or  zip file content encoded with base64